_posts/en/newsletters/2020-12-16-newsletter.md

```diff
@@ -0,0 +1,331 @@
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:harding to update
+
+- [Bitcoin Core 0.21.0rc3][Bitcoin Core 0.21.0] is a release candidate
+  for the next major version of this full node implementation and its
+  associated wallet and other software.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #20564][] makes two changes to the way that Bitcoin Core
+  signals support for `addrv2` messages ([BIP155][]):
+
+  - Bitcoin Core will only negotiate support for `addrv2` messages with peers that signal
+    they are using p2p version 70016 or higher. This restriction isn't required by the BIP,
+    but release testing has revealed that some other implementation will disconnect
+    from Bitcoin Core if they receive any unknown message, including
+    `sendaddrv2`. This change may be reverted in future versions of Bitcoin Core,
+    so other implementations are advised to always tolerate unknown p2p messages.
+
+  - The `sendaddrv2` message will now be sent between the `version` and
+    `verack` message, as required by the latest version of BIP155. See [BIPs
+    #1043](#bips-1043) below for more information about that change to the BIP.
+
+    This PR was back-ported to [the latest V0.21 release candidate][Bitcoin
+    Core 0.21.0] in [Bitcoin Core #20612][].
+
+- [Bitcoin Core #19776][] updates the `getpeerinfo` RPC with two new
+  fields. `bip152_hb_to` indicates that we asked the peer to relay new
+  blocks to us by sending a a [BIP152][] compact block without waiting
+  to ask if we need that specific block, called High-Bandwidth (HB)
+  mode.   `bip152_hb_from` indicates the peer asked us to be one of
+  their high-bandwidth peers.  By default, each node selects [up to
+  3][hb peers] high-bandwidth compact block peers.  (Despite the name,
+  high-bandwidth doesn't use much bandwidth compared to legacy block
+  relay; its optimizations for fast relay of new blocks just uses more
+  bandwidth than BIP152's low-bandwidth mode.)
+
+- [Bitcoin Core #19858][] adds a new method for finding high-quality peers with
+  the goal of making [eclipse attacks][topic eclipse attacks] more difficult.
+  It opens an outbound connection every five minutes on average to a new peer
+  and syncs headers with it. If the peer tells the node about new blocks, the
+  node disconnects an existing block-relay-only peer and gives that connection
+  slot to the new peer; otherwise, the new peer is dropped.  Provided the node
+  knows at least one honest peer, this peer rotation should raise the cost of
```

Comment:
Perhaps "knows the address of at least one honest node". "knows one honest peer" might be interpreted as being connected to at least one honest peer.